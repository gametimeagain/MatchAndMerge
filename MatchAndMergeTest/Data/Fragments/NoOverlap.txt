I did
n't fail th
e test. I just fou
nd one hun
dred ways to d
o it wrong.