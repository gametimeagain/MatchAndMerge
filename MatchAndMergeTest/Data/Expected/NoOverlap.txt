I didn't fail the test. I just found one hundred ways to do it wrong.